coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -0,0 +1,61 @@
+from coalib.results.result_actions.ResultAction import ResultAction
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.results.Result import Result
+from coalib.results.Diff import Diff
+from os.path import exists
+from os.path import isfile
+import shutil
+
+
+class IgnoreResultAction(ResultAction):
+
+    SUCCESS_MESSAGE = "Ignore keyword added."
+
+    @staticmethod
+    def is_applicable(result, original_file_dict, file_diff_dict):
+        """
+        For being applicable, the result has to point to a number of files
+        that have to exist i.e. have not been previously deleted.
+        """
+
+        if not isinstance(result, Result) or len(result.affected_code) == 0:
+            return False
+
+        filenames = set(src.renamed_file(file_diff_dict)
+                        for src in result.affected_code)
+        return all(exists(filename) for filename in filenames)
+
+    def apply(self, result, original_file_dict, file_diff_dict, language: str,
+              coalang_dir: str=None, no_orig: bool=False):
+        """
+        Ignore result
```

Comment:
maybe we can make somehow more explicit without having more than three words total that this actually touches your source code